cmd/evm/internal/t8ntool/transition.go

```diff
@@ -247,8 +247,8 @@ func Transition(ctx *cli.Context) error {
 		return NewError(ErrorJson, fmt.Errorf("failed signing transactions: %v", err))
 	}
 	// Sanity check, to not `panic` in state_transition
-	if chainConfig.IsLondon(big.NewInt(int64(prestate.Env.Number))) {
-		if prestate.Env.BaseFee == nil && chainConfig.Scroll.BaseFeeEnabled() {
+	if chainConfig.IsBanach(big.NewInt(int64(prestate.Env.Number))) {
+		if prestate.Env.BaseFee == nil {
```

Comment:
what if a block is > LondonBlock but < BanachBlock? we don't check `prestate.Env.BaseFee == nil && chainConfig.Scroll.BaseFeeEnabled()` for it any more?

Reply:
`BaseFeeEnabled` is removed (replaced by `IsBanach`). So

```go
chainConfig.IsLondon(big.NewInt(int64(prestate.Env.Number))) && chainConfig.Scroll.BaseFeeEnabled()
```

is equivalent to

```go
chainConfig.IsBanach(big.NewInt(int64(prestate.Env.Number)))
```